cluster/examples/kubernetes/ceph/object.yaml

```diff
@@ -56,6 +56,30 @@ spec:
     instances: 1
     # The affinity rules to apply to the rgw deployment or daemonset.
     placement:
+      podAntiAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
```

Comment:
We just want preferred antiaffinity, not required, right?

Reply:
I followed the same pattern for mds, do we have to change it too?

Reply:
I see, the required antiaffinity is for hosts and preferred for zones. With mds there are only two daemons, and with rgw the required antiaffinity could be a problem if you had more rgw instances than nodes. I'd suggest in the rgw example we just have preferred antiaffinity for hosts. Someone can always change from the example if they want.